README.md

```diff
@@ -56,3 +56,454 @@ The `pages/api` directory is mapped to `/api/*`. Files in this directory are tre
 ## Deploy on Vercel
 
 Coming soon
+
+
+## **SDK Usage**
+
+
+### **All Pools**
+
+#### Pools List (Get All Pools On Current Chain)
+
+import { usePoolsData } from '@hooks/usePoolsData';
+
+//returns all pools on Midas & Relevant Data
+const { data: allPools } = usePoolsData()
```

Comment:
This doesn't really showcases how to use the SDK itself, just how to use the hook. I'd say we document the sdk call itself (`getFusePoolData` or whatever is used inside the hook).